dandiapi/api/doi.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from typing import Dict
+
+from django.conf import settings
+import requests
+
+from dandiapi.api.models import Version
+
+
+def _generate_doi_data(version: Version) -> Dict:
+    prefix = settings.DANDI_DOI_API_PREFIX
+    dandiset_id = version.dandiset.identifier
+    version_id = version.version
+    doi = f'{prefix}/{dandiset_id}/{version_id}'
```

Comment:
Does it show for all 3?

Reply:
what?

An example DOI generated from this is `10.80507/000001/0.210323.2052`.